2016/ideas-list-swc.md

```diff
@@ -200,32 +221,18 @@ Skills required to complete the project (you can apply without some of them):
   for storing the results.
 * Knowledge of [Git][] for storing your work on [GitHub][]
 
-Any of these skills could be learned during the project and I will help you
+Any of these skills could be learned during the project and we will help you
 with them, but you probably can't learn *all* of them during GSOC.
 
-### Mentors
-
-* [@pbanaszkiewicz](https://github.com/pbanaszkiewicz)
-  (timezone: UTC+1h [Poland])
-* [@rgaiasc](https://github.com/rgaiacs)
-  (timezone: UTC+0h [UK])
-
-### Interested in making this project? Here's what you should do
-
-If you want to show interest in making this project, we expect you to do
-a demo application. You'll be able to use it as a base for you project further
-down the road.
-
-Your demo should consist of client and web server. The client should be able to
-send some information (like platform used, or perhaps versions of software
-installed) to the server, and server should be able to store it in the
-database.
+### First steps
 
-You can use [python-requests][] for the client-side and [Flask][] and
-[SQLite][] for the server part.
+Have a look at [Software Carpentry][swc]'s [installation-test scripts][swc-install]
+and make suggestions of how to improve it.
 
-Please store your demo on [GitHub][]. Finally, please contact us and let
-us know the link to your repository.
+A demo that consist of client and web server
+where the client sends the platform used to the server,
+and server stores it in the database
+will give you a bonus.
```

Comment:
I want to avoid saying 'something will give you a bonus' because it means if someone does the demo they'll be marked higher. I think they *probably* will, because we'll have more data to base our opinions.

Anyway, here's my proposition:
```markdown
As a bonus, you can implement
a demo consisting of a client and web server,
where the client sends platform (e.g. operating system)
details to the server, and servers stores that in the database.
```